zone/client_packet.cpp

```diff
@@ -14510,157 +14550,108 @@ void Client::Handle_OP_TargetCommand(const EQApplicationPacket *app)
 	if (g && g->HasRole(this, RolePuller))
 		g->SetGroupPullerTarget(GetTarget());
 
-	// For /target, send reject or success packet
-	if (app->GetOpcode() == OP_TargetCommand) {
-		if (GetTarget() && !GetTarget()->CastToMob()->IsInvisible(this) && (DistanceSquared(m_Position, GetTarget()->GetPosition()) <= TARGETING_RANGE*TARGETING_RANGE || GetGM())) {
-			if (GetTarget()->GetBodyType() == BT_NoTarget2 || GetTarget()->GetBodyType() == BT_Special
-				|| GetTarget()->GetBodyType() == BT_NoTarget)
-			{
-				//Targeting something we shouldn't with /target
-				//but the client allows this without MQ so you don't flag it
-				auto outapp = new EQApplicationPacket(OP_TargetReject, sizeof(TargetReject_Struct));
-				outapp->pBuffer[0] = 0x2f;
-				outapp->pBuffer[1] = 0x01;
-				outapp->pBuffer[4] = 0x0d;
-				if (GetTarget())
-				{
-					SetTarget(nullptr);
-				}
-				QueuePacket(outapp);
-				safe_delete(outapp);
-				return;
-			}
-
-			QueuePacket(app);
-
+	if (GetTarget())
+	{
+		if (GetGM())
+		{
 			GetTarget()->IsTargeted(1);
-			SendHPUpdate();
```

Comment:
Why the removal of the HP update? This was here for a reason

Reply:
Not removed.  Again, diff is confused by indentation, etc.

Reply:
No it is definitely removed

![image](https://github.com/EQEmu/Server/assets/3319450/2baad793-88d7-443c-9aee-b81fb7d8dde3)

Reply:
Nevermind. It wasn't there in master. 

My suggestion in the future is to not do indentation / formatting changes with stuff like this as it makes it very difficult to understand what is actually changed. 

Formatting changes to this extent should be separate from logic changes.

Reply:
Makes sense.